src/payment/strategies/googlepay/googlepay-payment-processor.spec.ts

```diff
@@ -238,6 +238,20 @@ describe('GooglePayPaymentProcessor', () => {
             expect(googlePayInitializer.parseResponse).toHaveBeenCalled();
         });
 
+        it('handles success response without last name', async () => {
+            jest.spyOn(googlePayInitializer, 'parseResponse').mockReturnValue(tokenizePayload);
+            jest.spyOn(requestSender, 'post').mockReturnValue(Promise.resolve());
+            jest.spyOn(store.getState().billingAddress, 'getBillingAddress').mockReturnValue(getGooglePayAddressMock());
+            const googlePaymentDataMock = getGooglePaymentDataMock();
+            googlePaymentDataMock.paymentMethodData.info.billingAddress = getGooglePayAddressMock();
```

Comment:
Why are we patching a mock?

Could `getGooglePaymentDataMock` not contain a billing address by default?

If not, maybe it's worth having two different mock generating function with more specific names